custom/icds_reports/utils/aggregation_helpers/delivery_forms.py

```diff
@@ -26,15 +26,18 @@ def aggregation_query(self):
 
         return """
         INSERT INTO "{tablename}" (
-          case_id, state_id, month, latest_time_end_processed, breastfed_at_birth, valid_visits
+          case_id, state_id, month, latest_time_end_processed, breastfed_at_birth, valid_visits,
+          where_born
         ) (
           SELECT
             DISTINCT case_load_ccs_record0 AS case_id,
             %(state_id)s AS state_id,
             %(month)s::DATE AS month,
             LAST_VALUE(timeend) over w AS latest_time_end_processed,
             LAST_VALUE(breastfed_at_birth) over w as breastfed_at_birth,
-            SUM(CASE WHEN (unscheduled_visit=0 AND days_visit_late < 8) OR (timeend::DATE - next_visit) < 8 THEN 1 ELSE 0 END) OVER w as valid_visits
+            SUM(CASE WHEN (unscheduled_visit=0 AND days_visit_late < 8) OR
+            (timeend::DATE - next_visit) < 8 THEN 1 ELSE 0 END) OVER w as valid_visits,
```

Comment:
This will be a bit easier to read if you indent this line so that its clear its part of the previous line's calculation

Reply:
DOne